Encryption.md

```diff
@@ -128,6 +127,26 @@ unique nonce with a length of 12 bytes (96 bits). Notice: the NIST
 specification uses a term “IV” for what is called “nonce” in the Parquet encryption design.
 
 
+#### 4.1.4 Invocation limit
+According to the section 8.3 of the NIST SP 800-38D document, one key can be used for up to 
```

Comment:
Can we add a link to this document, for easier reference?

Reply:
The link can change (already looks like a redirection). This doc can always be found by its name from the official NIST site.